curlify.py

```diff
@@ -5,7 +5,7 @@ def to_curl(request):
     headers = ["'{0}: {1}'".format(k, v) for k, v in request.headers.items()]
     headers = " -H ".join(sorted(headers))
 
-    command = "curl -X {method} -H {headers} -d '{data}' '{uri}'".format(
+    command = "curl -X {method} -H {headers} -d '{data}' '{uri}' --compressed".format(
```

Comment:
I think it's better to do it conditionally. So the function `to_curl` will accept optional parameter `compressed` and base on its value add or not add `--compressed` flag.

Because it can have unexpected effects: ```If this option is used and the server sends an unsupported encoding, curl will report an error.```

Reply:
Should I effect the change you advised and push it again?

Reply:
Yes, please